ironfish-rust/src/transaction/mod.rs

```diff
@@ -319,38 +326,19 @@ impl ProposedTransaction {
         change_goes_to: Option<PublicAddress>,
         intended_transaction_fee: u64,
     ) -> Result<Transaction, IronfishError> {
-        let mut change_notes = vec![];
+        let public_address = spender_key.public_address();
 
-        for (asset_id, value) in self.value_balances.iter() {
-            let is_native_asset = asset_id == &NATIVE_ASSET;
-
-            let change_amount = match is_native_asset {
-                true => *value - i64::try_from(intended_transaction_fee)?,
-                false => *value,
-            };
-
-            if change_amount < 0 {
-                return Err(IronfishError::new(IronfishErrorKind::InvalidBalance));
-            }
-            if change_amount > 0 {
-                let change_address = change_goes_to.unwrap_or_else(|| spender_key.public_address());
-                let change_note = Note::new(
-                    change_address,
-                    change_amount as u64, // we checked it was positive
-                    "",
-                    *asset_id,
-                    spender_key.public_address(),
-                );
+        let signed_fee = i64::try_from(intended_transaction_fee)?;
```

Comment:
Multiple meanings of signed and unsigned!

I can't think of a name that I like better, but this could cause some confusion

Reply:
Yea thats fair, what about `i64_fee`? I will update